### ISSUE TITLE
Typogrify breaks on empty string

### ISSUE THREAD

**Issue by chasegiunta** [nystudio107/craft3-typogrify#18]
I'm applying the `typogrify` filter on a typical body field, but this for body fields that don't have any content entered yet, thus give an empty string, typogrify throws a Type Error

```
TypeError
Return value of nystudio107\typogrify\variables\TypogrifyVariable::normalizeText() must be of the type string, null returned
```

A work-around is to check for the length on the body field, but this seems like unnecessary/abnormal behavior for a filter running on a empty string. (like `nl2br`, `upper`, `capitalize` etc.)

**Comment by khalwat** [comment]
hrm yeah probably it should be able to handle `null` being passed in, and just return an empty string or something

**Comment by khalwat** [comment]
Fixed in 1.1.15: https://github.com/nystudio107/craft-typogrify/releases/tag/1.1.15
